server/src-lib/Hasura/RQL/Types/EventTrigger.hs

```diff
@@ -218,3 +220,17 @@ data InvokeEventTriggerQuery
   } deriving (Show, Eq, Lift)
 
 $(deriveJSON (aesonDrop 4 snakeCase){omitNothingFields=True} ''InvokeEventTriggerQuery)
+
+data PauseEventTriggerQuery
+  = PauseEventTriggerQuery
+  { petqName  :: !TriggerName
+  } deriving (Show, Eq, Lift)
+
+$(deriveJSON (aesonDrop 4 snakeCase){omitNothingFields=True} ''PauseEventTriggerQuery)
+
+data ResumeEventTriggerQuery
+  = ResumeEventTriggerQuery
+  { retqName  :: !TriggerName
+  } deriving (Show, Eq, Lift)
+
+$(deriveJSON (aesonDrop 4 snakeCase){omitNothingFields=True} ''ResumeEventTriggerQuery)
```

Comment:
I realize the other query types are defined in this module, so you’re just doing what the existing code does, but I don’t think there’s any reason we can’t define these alongside the functions that handle them, in `Hasura.RQL.DDL.EventTrigger`. Keeping the types close to the functions that need them is always better if possible in my opinion! So I’d move these definitions into that module, and while you’re at it, maybe you can move the other query types there, too.